### PR TITLE
Use new documentation layout on Vanilla site

### PR DESCRIPTION
**Pull request by bartaz**
## Done

Apply new documentation layout to Vanilla website.
This mostly affects docs section, but the whole site now uses paper background as well.

Fixes [WD-6055](https://warthogs.atlassian.net/browse/WD-6055)

## QA

- Open [demo](https://vanilla-framework-4863.demos.haus/docs)
- Review the docs, make sure layout is fine and components look good on new background

[WD-6055]: https://warthogs.atlassian.net/browse/WD-6055?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
Looks great, and the accordion helps a lot I think. One thing - resizing from smaller width to bigger leaves the codepen examples at the earlier height, with a lot of white space underneath. Can we resize them onresize? 
image

**Comment by bartaz** [comment]
> Looks great, and the accordion helps a lot I think. A few niggles I noticed:
> 
> * clicking on any sidenav item leads to the scrollbar becoming visible for a fraction of a second

I can't reproduce it and don't really know what we could do about it. If it is a visible issue we can address it separately.

> * resizing from smaller width to bigger leaves the codepen examples at the earlier height, with a lot of white space underneath. Can we resize them onresize?

Done.

> * the sidenav highlight goes under the scrollbar

macOS shows scrollbar over the content, not much we can do about it.

> * the footer should use the dark theme link colours:

Done

> * I know we have a separate issue for this, but just pointing out it is still protruding. We discussed making menu item padding same as grid gutter padding to solve this. Can it be a driveby here or something we merge before this pr?

I made a quick hack for it (shifted the items to make them aligned). Adjusting padding is a separate issue and it turned out not to be quick (because of legacy of old navigation styles), so I'll address it separately later.

> * we should be using the paper colour to stay consistent with the new branding

Done

File: templates/_layouts/docs.html [hunk 1]
```diff
@@ -9,84 +11,117 @@
...
+            {% block content_title %}
+
+              {% if title %}
                 <h1>{{ title.split('|')[0] }}</h1>
-              </div>
               {% endif %}
 
               {% include "_layouts/_component_tabs.html" %}
+              {% if not page_tabs %}
+              <hr>
```
[the remark this answers is not in the document]
Comment (akbarkz) [reply templates/_layouts/docs.html:91]: And everywhere else where there is a similar element

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+
+{% block style %}
+<style>
+  body { margin: 0; }
+</style>
+{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:12]: why two line breaks?

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+<div class="l-docs">
+  <div class="l-docs__header">
+    <header id="navigation" class="p-navigation is-dark">
+      <div class="l-docs__container">
+        <div class="l-docs__sidebar">
+          <div class="p-navigation__banner">
+            <div class="p-navigation__tagged-logo">
+              <a class="p-navigation__link" href="#">
+                <div class="p-navigation__logo-tag">
+                  <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:23]:
```suggestion
                  <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="" />
```

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 4]
```diff
@@ -0,0 +1,422 @@
...
+      </div>
+      <div class="l-docs__main">
+        <div class="row">
+          <form class="p-search-box u-no-margin--bottom">
+            <input type="search" class="p-search-box__input" name="q" placeholder="Search documentation" required="" autocomplete="on"/>
+            <button type="reset" class="p-search-box__reset" name="close"><i class="p-icon--close">Close</i></button>
+            <button type="submit" class="p-search-box__button" name="submitSearch"><i class="p-icon--search">Search</i></button>
+          </form>
+        </div>
+        <!--div class="row grid-demo">
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:68]: is this commented part really needed? if so, could we add a comment explaining why do we still want to keep it?

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+      <div class="p-side-navigation__drawer">
+        <div class="p-side-navigation__drawer-header">
+          <a href="#" class="p-side-navigation__toggle--in-drawer js-drawer-toggle" aria-controls="drawer">
+            Toggle side navigation
+          </a>
+        </div>
+        <h3>Side navigation</h3>
+        <ul>
+          <li>
+            <a>First page</a>
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:131]: href here too?

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 6]
```diff
@@ -0,0 +1,422 @@
...
+              <li><a class="p-table-of-contents__link" href="#link4">Service statuses</a></li>
+            </ul>
+          </nav>
+        </div>
+      </aside>
+
+  </aside>
+
+  <main class="l-docs__main">
+    <!-- div class="p-strip is-shallow">
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:205]: also here, let's remove it if not needed or add a comment

File: templates/docs/examples/layouts/documentation-full-width.html [hunk 7]
```diff
@@ -0,0 +1,422 @@
...
+      <div class="l-docs__sidebar">
+        <p style="padding-left: 1.5rem">© 2020 Canonical Ltd.</p>
+      </div>
+      <div class="l-docs__main">
+        <nav class="row" aria-label="Footer">
+          <div class="has-cookie">
+            <p><a class="is-dark" href="#">Ubuntu</a> and <a class="is-dark" href="#">Canonical</a> are registered trademarks of Canonical Ltd.</p>
+            <ul class="p-inline-list--middot">
+              <li class="p-inline-list__item">
+                <a class="is-dark" href="#"><small>Legal information</small></a>
```
Comment (akbarkz) [inline templates/docs/examples/layouts/documentation-full-width.html:402]: why do we leave links that don't lead anywhere? My expectation would be to have read addresses to take users to when they click on these links.

File: templates/docs/examples/layouts/documentation-full-width.html (hunk 4 above)
Comment (bartaz) [reply templates/docs/examples/layouts/documentation-full-width.html:68]: No, it's leftover from testing. Thanks.

**Review by akbarkz — APPROVED** [review]

**Pull request merged**